index.html

```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card3">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card4">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card5">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card6">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+      </section>
+    </section>
+
+    <section class="about_me">
+      <section class="description">
+        <h1>About me</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </section>
+      <section class="resume">
+        <button>Get My Resume</button>
+        <a class="about_bgimg"></a>
```

Comment:
- [ ] Background images are best on the page using CSS property `background-image`  and not with an element as you have use here. Kindly remove this `<a class="about_bgimg"></a>`.